dags/test_slack_notifier.py

```diff
@@ -0,0 +1,40 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.python_operator import PythonOperator
+
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Test if the Slack notifier is working",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+
+def task_fail():
+    raise Exception("Task failed")
+
+
+with DAG(
+    dag_id=f"test_slack_notifier_{DEPLOYMENT_ENVIRONMENT}",
+    default_args=default_args,
+    schedule_interval="*/5 * * * *",
```

Comment:
According to [the docs](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dag-run.html#cron-presets) we can set the schedule to be `None`, so that the DAG can only be manually triggered. That probably fits the intent of this DAG, to only be triggered as a test?

Reply:
i agree with this to avoid spamming the Airflow channel 📥